src/Components/Header/Header.css

```diff
@@ -30,41 +30,48 @@
 .logo-name  {
   font-weight: 700;
   font-size: 1.25em;
-  margin-left: 5px;
+  margin: 0% 0% .5% 1%;
+}
+
+.right-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  width: 50%;
 }
 
 .toggle-container  {
   display: flex;
   flex-direction: row;
   align-items: center;
   gap: 1vw;
-  border: 1.5px #767676 solid;
-  border-radius: 25px;
-  padding: 3px;
-  height: 35px;
-  width: 220px;
+  border: .1em #767676 solid;
+  border-radius: 2em;
+  padding: 1% 1.5% 1% 1.5%;
+  height: 35%;
+  width: fit-content;
```

Comment:
I noticed that the toggle container contains the button when hovered and clicked now. Not sure if this is the line of code, but nice!